### PR TITLE
Front end board development

### PR DESCRIPTION
**Pull request by edlingao**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,5 +1,74 @@
+import {Player} from './player.js';
+import {Game} from './game.js';
 let play = document.querySelector('#start-game');
+let cells  = document.querySelectorAll('.cell');
+let label = document.querySelector('#player-name');
+let replay = document.querySelector('#replay');
+let formContainer = document.querySelector('#hidden');
+let winMessage = document.querySelector('#winner');
```
Comment (azdrenymeri) [inline js/main.js]: You are polluting the global object.
Encapsulate these statements inside the functions or factories that these variables are being used.

File: js/main.js [hunk 2]
```diff
@@ -1,5 +1,74 @@
...
+let player2;
+let arrBoard = ['', '', '', '', '', '', '', '', ''];
+let game = Game(arrBoard);
+
+function addSymbol(cell,player ,symbol){
+    cell.innerHTML = `<${player}>${symbol}</${player}>`;
+}
+function toggleSymbols(toggle){
+    return toggle ? "X" : "O"
+}
```
Comment (azdrenymeri) [inline js/main.js]: `Optional` Use the ES6 syntax to declare these two functions

Comment (edlingao) [reply js/main.js]: Can you show me the errors on your console? i can play, im not sure why is not working on you

**Comment by edlingao** [comment]
Can you show me the errors on your console? i can play, im not sure why is not working on you

**Comment by edlingao** [comment]
Are you trying to run it localy? Chrome blocks every local js files, please use it with LiveServer to run it.

**Pull request merged**